src/Service.Host/client/src/components/POReqs/POReqUtility.js

```diff
@@ -844,7 +832,7 @@ function POReqUtility({ creating }) {
                             <Grid item xs={4}>
                                 <InputField
                                     fullWidth
-                                    value={req.totalReqPrice}
+                                    value={parseFloat(req.totalReqPrice).toFixed(2)}
```

Comment:
why do we have to parse to a float from a number field?

Reply:
was just using the toFixed(2) to force it to always show 2 decimal places even when pulled straight from the db - Steph asked if it could display 2 decimal places for this, carriage and unit price so sort of did this one as a compromise (to fixed on the others messes up typing but this one is disabled and calculated from others). Maybe better leaving them consistent and removing this again? She also wanted commas for big numbers but think going to leave that for now unless she asks again

Reply:
I might put it on my list to look at inputField component to try add a comma formatting option and fixed no of decimal places for numbers, but doesn't seem the most important thing right now